packages/@sanity/util/test/PathUtils.test.ts

```diff
@@ -24,7 +24,8 @@ test('fromString: throws if not a string', (t) => {
   t.throws(() => fromString(), 'Path is not a string')
   // @ts-ignore
   t.throws(() => fromString(13), 'Path is not a string')
-  t.throws(() => fromString(null as any), 'Path is not a string')
+  // @ts-ignore
```

Comment:
Why not `// @ts-expect-error` instead of `// @ts-ignore`?

Reply:
I copied what was used elsewhere in the file, for consistency.